test/testcafe/spec/ChallengePIV_spec.js

```diff
@@ -0,0 +1,81 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+import PIVPageObject from '../framework/page-objects/PIVPageObject';
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import xhrIdentifyWithThirdPartyIdps from '../../../playground/mocks/data/idp/idx/identify-with-third-party-idps';
+import xhrVerifyWithPIVOnly from '../../../playground/mocks/data/idp/idx/identify-with-piv-only.json';
+import xhrVerifyPIVError from '../../../playground/mocks/data/idp/idx/error-identify-with-piv.json';
+
+const logger = RequestLogger(/introspect/,
+  {
+    logRequestBody: true,
+    stringifyRequestBody: true,
+  }
+);
+
+
+const verifyWithSelectPIVIdpMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithThirdPartyIdps)
+  .onRequestTo('https://okta.mtls.okta.com/sso/idps/mtlsidp?stateToken=eyJ6aXAiOiJERUYiLCJhbGlhcyI6ImVuY3J5cHRpb25')
+  .respond('<html><h1>MTLS Url for PIV auth</h1></html>');
+
+const verifyWithOnlyPIVIdPMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrVerifyWithPIVOnly)
+  .onRequestTo('https://okta.mtls.okta.com/sso/idps/mtlsidp?stateToken=eyJ6aXAiOiJERUYiLCJhbGlhcyI6ImVuY3J5cHRpb25')
+  .respond('<html><h1>MTLS Url for PIV auth</h1></html>');
+
+const verifyErrorMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrVerifyPIVError);
+
+async function setup(t, isVerify) {
+  const identityPage = new IdentityPageObject(t);
+  await identityPage.navigateToPage();
+
+  const { log } = await t.getBrowserConsoleMessages();
```

Comment:
let's use `checkConsoleMessages` instead if possible
```
import { checkConsoleMessages } from '../framework/shared';
```

Reply:
👍 sorry forgot this